### PR TITLE
[16.0][ADD] account_reconcile_analytic_tag: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

ejemplo

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT51885

### CODE REVIEW

File: account_reconcile_analytic_tag/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Account reconcile analytic tag",
```
Comment (pedrobaeza) [inline account_reconcile_analytic_tag/__manifest__.py]:
```suggestion
    "name": "Analytic tags in account reconciliation",
```

File: account_reconcile_analytic_tag/models/account_bank_statement_line.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = "account.bank.statement.line"
+
+    analytic_tag_ids = fields.Many2many(
```
Comment (pedrobaeza) [inline account_reconcile_analytic_tag/models/account_bank_statement_line.py]: Shouldn't it be this name following current fields nomenclature?

```suggestion
    manual_analytic_tag_ids = fields.Many2many(
```

**Review by pedrobaeza — COMMENTED** [review]
Tested and working. Just a few code remarks

File: account_reconcile_analytic_tag/models/account_bank_statement_line.py (hunk 2 above)
Comment (victoralmau) [reply account_reconcile_analytic_tag/models/account_bank_statement_line.py]: The analytical distribution field does not have the manual prefix but if the name is consistent, it is already changed.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Functional tests :+1:

**Pull request merged**